crates/rooch-types/src/transaction/mod.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use ethereum::{EthereumSignature, EthereumTransactionData};
+
+pub mod ethereum;
+
+pub enum TransactionType {
+    Ethereum,
+    Move,
```

Comment:
Need to distinguish transactions from different Move chains?

Reply:
Just use an example for support multichain transactions, it is not finished yet.